examples/get-started/pure-js/carto/app.js

```diff
@@ -1,20 +1,47 @@
 import maplibregl from 'maplibre-gl';
 import {Deck} from '@deck.gl/core';
-import {fetchMap} from '@deck.gl/carto';
+import {CartoLayer, setDefaultCredentials, BASEMAP, MAP_TYPES} from '@deck.gl/carto';
 
-const cartoMapId = 'ff6ac53f-741a-49fb-b615-d040bc5a96b8';
+setDefaultCredentials({
+  apiBaseUrl: 'https://gcp-us-east1-11.dev.api.carto.com',
+  accessToken: 'XXX'
+});
+
+// Add basemap for context.
+const map = new maplibregl.Map({container: 'map', style: BASEMAP.VOYAGER, interactive: false});
 
-// Get map info from CARTO and update deck
-fetchMap({cartoMapId}).then(({initialViewState, mapStyle, layers}) => {
-  const deck = new Deck({canvas: 'deck-canvas', controller: true, initialViewState, layers});
+new Deck({
+  canvas: 'deck-canvas',
 
-  // Add Mapbox GL for the basemap. It's not a requirement if you don't need a basemap.
-  const MAP_STYLE = `https://basemaps.cartocdn.com/gl/${mapStyle.styleType}-gl-style/style.json`;
-  const map = new maplibregl.Map({container: 'map', style: MAP_STYLE, interactive: false});
-  deck.setProps({
+  initialViewState: {latitude: 55.5, longitude: 12, zoom: 7},
+  controller: true,
+
+  ...(map && {
+    mapStyle: BASEMAP.VOYAGER,
     onViewStateChange: ({viewState}) => {
       const {longitude, latitude, ...rest} = viewState;
       map.jumpTo({center: [longitude, latitude], ...rest});
     }
-  });
+  }),
+
+  layers: [
+    new CartoLayer({
+      connection: 'deb-bigquery',
+      type: MAP_TYPES.RASTER,
+      data: 'cartodb-data-engineering-team.jarroyo_raster.hillshade1x1_quadbin_2',
```

Comment:
This dataset is not public, we need to move it cartobq.testtables.

Reply:
Also, we need to make sure this raster has the resolution metadata

Reply:
None of this file is to be commited, it's just for testing - I left a comment before